src/controllers/PhotoController.ts

```diff
@@ -16,6 +16,19 @@ class PhotoController implements IController {
     const photo = await this.photoService.create(photoPayload);
     res.status(201).json({ data: photo });
   };
+
+  public updateById: IRequestHandler = async (req, res) => {
+    const photoId = Number(req.params.id);
+    const photoPayload = req.body;
+    const photo = await this.photoService.updateById(photoId, photoPayload);
+    res.status(201).json({ data: photo });
+  };
+
+  public deleteById: IRequestHandler = async (req, res) => {
+    const photoId = Number(req.params.id);
+    const photo = await this.photoService.deleteById(photoId);
+    res.status(201).json({ data: photo });
```

Comment:
201 Status code should be used for created only, in this case we should use 204.